scripts/translation/lib/RevtagParser.php

```diff
@@ -57,13 +54,16 @@ public static function parseComment( DOMNode $node , RevtagInfo & $ret , $filena
 
         if ( str_starts_with( $text , "EN-" ) )
         {
+            // /EN-Revision:\s*(\S+)\s*Maintainer:\s*(\S+)\s*Status:\s*(\S+)/ // restric maintainer with no spaces
```

Comment:
"restrict" ?

Reply:
Yes, fixed. As this is clearing for merging, there is translations with user names with spaces?

These comments are effectually a TODO, before implementation.

Reply:
I'm not sure if there are, I can run this on all of the translations to check.